server/tests/geocodes.test.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-restricted-syntax, no-await-in-loop */
+const { v4 } = require('uuid');
+const { startDB, closeDB } = require('./util/db');
+const { getUserSites, makeUser } = require('../services/user.js');
+const { saveSites } = require('../services/employers.js');
+const { makeParticipant, getParticipants } = require('../services/participants.js');
+const { getParticipantCoords } = require('../services/geocodes.js');
+
+describe('Geocodes Service', () => {
+  beforeAll(async () => {
+    await startDB();
+  });
+
+  afterAll(async () => {
+    await closeDB();
+  });
+
+  // Tests
+  // Add participant, query for coords and compare
+  // Update participant postal code, query for new coords
+  // Add site, query for coords and compare
+  // Update site postal codes, query for new coords
+
+  it('Adds and updates a participant postal code, checking for coords', async () => {
+    const participant = {
+      maximusId: 648690,
+      lastName: 'Extra',
+      firstName: 'Eddy',
+      postalCode: 'V0A2V3',
+      postalCodeFsa: 'V1V',
+      phoneNumber: '2502223333',
+      emailAddress: 'eddy@example.com',
+      interested: 'yes',
+      nonHCAP: 'yes',
+      crcClear: 'yes',
+      preferredLocation: 'Fraser',
+    };
+
+    const res = await makeParticipant(participant);
+    const coords = await getParticipantCoords(res.id);
+
+    console.log(coords);
```

Comment:
Is this log + the commented code intended to be included?

Reply:
Shouldn't there be an `expect` somewhere in here?

Reply:
Yep! Never finished writing this test, apparently 😅